loadtest.yml

```diff
@@ -0,0 +1,35 @@
+config:
+  target: 'http://localhost:4000'
+  phases:
+    - duration: 30
+      arrivalRate: 5
+      name: "Warm up"
+    - duration: 60
+      arrivalRate: 5
+      rampTo: 100
+      name: "Ramp up to 100rps"
+    - duration: 60
+      arrivalRate: 100
```

Comment:
interesting seeing how many phases you used. Did you notice any change in the max rps you were able to achieve with this slower build up?